extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/descr

```diff
@@ -0,0 +1,6 @@
+DRAFT Mathematical Components Library on Analysis
+
+This library contains a DRAFT and UNSTABLE embryo of an analyis
```

Comment:
How do you say,... "ceinture & bretelles" ?
For the title, I'd say "Mathematical Components Library on Analysis (DRAFT)", otherwise it looks like a beer to me ;-)

Are there axioms? Does it use's stdlib axioms? (they should be mentioned)

Reply:
Thanks for the comments @gares. I will update my PR accordingly.